grafana/dashboards/stratagem-dashboard-1.json

```diff
@@ -0,0 +1,168 @@
+{
+  "annotations": {
```

Comment:
What is the purpose of annotations?

Reply:
This was generated by grafana. From their documentation: 
> Annotations provide a way to mark points on the graph with rich events. When you hover over > an annotation you can get event description and event tags. The text field can include links to > other systems with more detail.